src/main/resources/seed/db/migration/R__Seed_Data.sql

```diff
@@ -44,3 +45,6 @@ VALUES ('28e47d30-30bf-4dab-a8eb-9fda3f6400e8', '7fffcc6a-11f8-4713-be35-cf5ff1a
        ('d3abc217-75ee-46e9-a010-368f30282367', '7fffcc6a-11f8-4713-be35-cf5ff1aee510'),
        ('1811faa6-d568-4fc4-83ce-41118b90242e', '7fffcc6a-11f8-4713-be35-cf5ff1aee518'),
        ('1811faa6-d568-4fc4-83ce-41118b90242e', '7fffcc6a-11f8-4713-be35-cf5ff1aee519');
+
+INSERT INTO referral (referral_id, offering_id, prison_number, referrer_id)
+VALUES ('c11fab18-dc8d-420c-9c82-d0edd373732d', '20f3abc8-dd92-43ae-b88e-5797a0ad3f4b', 'ABC1234', '5105A589-75B3-4CA0-9433-B96228C1C8F3');
```

Comment:
Is the referrer id here based on something real or a random UUID? Should it be lowercase?

Reply:
It's `AUTH_ADM`'s ID: https://github.com/ministryofjustice/hmpps-auth/blob/4464bc3f7909cf95c847f9a975410e5fd09ffecc/src/main/resources/db/dev/data/auth/V900_3__users.sql#L87

Interestingly it's upper case there, but when logging the `userId` in the UI it's lower case - will lower case it here

Reply:
I've updated the casing and added more details on where these data come from in the commit message